File: Assets/Scripts/Entities/PostItVal.cs
public class PostItVal : AbsMessage
{
    public int id { get; set; }
    public bool isSelected { get; set; }
    public int currentPostItState { get; set; }
    public float dwellTimeRatio { get; set; }
    public float saccadeInRatio { get; set; }
}

File: PythonScripts/Scripts/MainPLMS/Entity/postItVal.py
class PostItVal(object):
    def __init__(self, id, isSelected, currentPostItState, dwellTimeRatio, saccadeInRatio):
        self.id = id
        self.isSelected = isSelected
        self.currentPostItState = currentPostItState
        self.dwellTimeRatio = dwellTimeRatio
        self.saccadeInRatio = saccadeInRatio
        # need saccade in data

    @classmethod
    def from_json(cls, data):
        return cls(**data)
